### PR TITLE
add imags and steps for accessing Open Firmware with Coreboot

### PR DESCRIPTION
**Pull request by ahoneybun**
adds images and steps for the new Open Firmware with Coreboot for the galp4 and darp6.

### CODE REVIEW

File: _articles/boot-menu.md [hunk 1]
```diff
@@ -19,3 +19,19 @@ To enter BIOS for recent System76 laptops, hold the <kbd>F2</kbd> key immediatel
 To enter BIOS for desktops (besides the Meerkat), hold down the <kbd>DEL</kbd> key.  To show the boot menu, hold the <kbd>F8</kbd> or <kbd>F12</kbd> key. 
 
 For the Meerkat, hold the <kbd>F2</kbd> key for the BIOS, or the <kbd>F10</kbd> key for the boot menu.
+
+The newest Darter Pro (darp6) and Galago Pro (galp4) ship with Open Firmware with Coreboot so their menu system looks different then our other laptops. The BIOS can be accessed by rebooting the laptop and tapping the <kbd>ESC</kbd> key. The following screenshots show what the menus look like.
```
Comment (brs17) [inline _articles/boot-menu.md]: By saying `The newest Darter Pro` you are requiring the team to go in and update this on the next release. Perhaps instead just remove `newest`.

Comment (brs17) [inline _articles/boot-menu.md]: You should be using `than` instead of `then`.

File: _articles/boot-menu.md [hunk 2]
```diff
@@ -19,3 +19,19 @@ To enter BIOS for recent System76 laptops, hold the <kbd>F2</kbd> key immediatel
...
+
+### Boot Default
+
+![Main menu](/images/boot-menu/homepage.png)
```
Comment (brs17) [inline _articles/boot-menu.md]: I may be mistaken, but I thought I recalled @thomas-zimmerman saying he wanted to replace the qemu information with actual product information.

**Comment by jacobgkau** [comment]
I will make a separate PR for the corrections, not sure why this got merged.

File: _articles/boot-menu.md (hunk 1 above)
Comment (jacobgkau) [reply _articles/boot-menu.md]: Fixed in PR 238.

**Pull request merged**